### DISCUSSION TITLE
Disabling Hausdorff Measurement Gives Error

### DISCUSSION THREAD

**Discussion by GorkemP** [FETS-AI/Challenge#51] [General]
Dear Organizers,

After disabling the hausdorff measurement (`include_validation_with_hausdorff=False`) by using the recently committed project, it gives the following error, 

![image](https://user-images.githubusercontent.com/7123842/123439096-0cd7fb00-d5da-11eb-8c4d-8c986b461043.png)

Is there a way to fix this?

Thank you in advance

**Comment by sarthakpati** [comment]
Pinging @brandon-edwards and @psfoley

**Reply by brandon-edwards** [reply]
Hi @GorkemP,
Could you please try (while in your virtual environment):
```
pip uninstall fets 
pip uninstall openfl
```
Then while in `Challenge/Task_1`: run `pip install ./`

Try running the experiment again, and tell us what you see. I'm wondering whether you are not picking up the changes in fets that are needed for this new functionality.
Thanks.

**Reply by GorkemP** [reply]
Thank you for your suggestion, as you suspected I missed to install new packages after fetching the last commit. Now, it works.
Thanks.

**Reply by brandon-edwards** [reply]
Great, thanks for letting us know.
